src/LinSys/LinSysMerger.h

```diff
@@ -1,7 +1,7 @@
 // *****************************************************************************
 /*!
   \file      src/LinSys/LinSysMerger.h
-  \author    J. Bakosi
+  \author    Los Alamos National Laboratory
```

Comment:
We might want to take this offline, but I disagree that the original author will have the best understanding of why a code block written by twelve different people does what it does. This is why when you look at the Linux kernel you don't immediately send Linus an email about the methodology behind gamepad support.

Similarly, I make a lot of mistakes when I code (that is why I like code reviews and regression tests. I can force someone to double check me). git-blame is the tool to use when we find out that I added an off by one error under certain cases two years ago. The original author has no idea that I forgot what language I was in and might think it was intentional due to weird interloop dependencies. That is what git-blame is for and adding additional information just leads to delays if the wrong person is asked.

And, speaking more philosophically: Open source codes are a community project and can have any number of authors. I would prefer no author blocks at all because maybe down the line someone from a random University will be using quinoa and contribute a feature or a bug fix that some random postdoc who drew the short straw and got code maintenance responsibilities approved.
Github is nice because it provides a great list of all contributors and git-blame is great because it lets you figure out which of those were naughty when you spend a few hours debugging and nail down a problematic statement. And the original author may move on to bigger and better things and it might be some random person who is now in charge of code maintenance and approval of pull requests.

While more about copyrights (which are a necessary "evil" when you have open source projects that are owned and financially supported by an entity), https://opensource.com/law/14/2/copyright-statements-source-files provides a moderately well written argument on the subject.